README.md

```diff
@@ -1,9 +1,125 @@
 
-For instructions on how to set up Freesound for development, [see the wiki](https://github.com/MTG/freesound/wiki/Development-setup)
-
-For notes about development, see DEVELOPERS.md
+Freesound is a project by the [Music Technology Group](http://www.mtg.upf.edu) (MTG), [Universitat Pompeu Fabra](http://upf.edu) (UPF).
 
 [![Build Status](https://travis-ci.org/MTG/freesound.svg?branch=master)](https://travis-ci.org/MTG/freesound)
+
+
+## License
+
+All the source code in this repository is licensed as GNU Affero. Some of the dependencies might have their own licenses. See the [_LICENSE](https://github.com/MTG/freesound/tree/master/_LICENSE) folder for more details.
+
+
+## Authors
+
+For a list of authors please check out the [contributors](https://github.com/MTG/freesound/graphs/contributors) page.
+
+
+## Development
+
+Freesound is composed by a numnber of different services which can be run and orchestrated using Docker . The main service is provided by the `web` container which runs the Freesound Django application. Check out [this blog post](https://opensource.creativecommons.org/blog/entries/freesound-intro/) for some information about the Freesound technology stack. If you're to do development for Freesound, please checkout the [DEVELOPERS](https://github.com/MTG/freesound/blob/master/DEVELOPERS.md) file for some guidelines. 
+
+Below are instructions for setting up a local Freesound installation for development. It is assumed that you have a working Docker installation.
+
+
+### Setting up Freesound for development in 13 easy steps
+
+1. Clone source code repository
+
+  ```
+git clone git@github.com:MTG/freesound.git
+cd freesound
+```
+
+2. Create a directorty named `freesound-data` inside the repository folder
+
+  ```
+mkdir freesound-data
+```
+
+3. Download the Freesound development data zip file (~20GB) and uncompress it inside `freesound-data`. You should request this file to Freesound admins. File structure should look like this:
+
+  ```
+freesound/
+freesound/freesound-data/
+freesound/freesound-data/analysis/
+freesound/freesound-data/avatar/
+...
+```
+
+4. Download Freesound development similarity index data and tag recommendation models and place them under `freesound-data/similarity_index/` and `freesound-data/tag_recommendation_models` directories respectively (you'll need to create the directories). You should request these files to Freesound admins.
+
+
+5. Rename `freesound/local_settings.example.py` file so you can customise Django settings if needed and create a `.env` file with your local user UID
+
+  ```
+cp freesound/local_settings.example.py freesound/local_settings.py
+echo FS_USER_ID_FROM_ENV=$(id -u) > .env
+```
+
+6. [Optional] Create API credentials for the 3rd party services listed below and add them to your own `freesound/local_settings.py` file (check `settings.py` to know the config parameter names that you need to fill in):
```

Comment:
this could be improved in the future. I know that if we list settings here then it'll get out of date, but it's a bit weird to say "look in file x to see the settings names to copy to file y".
Perhaps we should have a section in local settings with a comment `# 3rd party services`, with a comment explaining how to register for this service and what values to add to the file

Reply:
agree, although these services are not really needed so for now we can leave it like that. Maybe the mapbox one could be of more interest